test/sqllogictest/dates-times.slt

```diff
@@ -751,6 +751,24 @@ SELECT (INTERVAL '1-3' YEAR TO MONTH)::text
 ----
 1 year 3 months
 
+query RRRRR
+SELECT EXTRACT(hour from TIME '11:12:42.666'),
+    EXTRACT(minute from TIME '11:12:42.666'),
+    EXTRACT(second from TIME '11:12:42.666'),
+    EXTRACT(milliseconds from TIME '11:12:42.666'),
+    EXTRACT(microseconds from TIME '11:12:42.666')
+----
+11 12 42.666 42666 42666000
+
+query error
```

Comment:
I much prefer to include some part of the error message to make the test more legible in the future.

```diff
diff --git a/test/sqllogictest/dates-times.slt b/test/sqllogictest/dates-times.slt
index 7d8323659..3eb8c473f 100644
--- a/test/sqllogictest/dates-times.slt
+++ b/test/sqllogictest/dates-times.slt
@@ -760,13 +760,13 @@ SELECT EXTRACT(hour from TIME '11:12:42.666'),
 ----
 11 12 42.666 42666 42666000
 
-query error
+query error unknown units 'year'
 SELECT EXTRACT(year from TIME '11:12:42.666')
 
-query error
+query error unknown units 'month'
 SELECT EXTRACT(month from TIME '11:12:42.666')
 
-query error
+query error unknown units 'day'
 SELECT EXTRACT(day from TIME '11:12:42.666')
 
 query RR
```

Reply:
Added the error messages.

Reply:
Just wanted to note that it's actually much more important than just legibility! If you don't include the error message, it's very easy to typo the test, e.g.:

```
query error
SELECT EXTRACT (YEAR FROM TIEM '11:12:42.666')
```

SLT will happily stamp that test as a success, but what you're really testing there is that `TIEM` is recognized as a typo. 

(...speaking as someone who's done this more than a few times.)